templates/docker/conda_install_devel_deps

```diff
@@ -9,7 +9,9 @@ RUN source activate rapids \
 
 # Install rapids-build-env from conda meta-pkg
 RUN ${RAPIDS_DIR}/utils/condaretry install -y -n rapids --freeze-installed \
-        rapids-build-env=${RAPIDS_CONDA_VERSION_SPEC}
+        rapids-build-env=${RAPIDS_CONDA_VERSION_SPEC} \
+    && conda remove -y -n rapids --force rapids-build-env \
```

Comment:
I assume we're uninstalling `rapids-build-env` because it includes an earlier version of numba ([src](https://github.com/rapidsai/integration/blob/branch-0.13/conda/recipes/rapids-build-env/meta.yaml#L71)). will we be updating `numba` in `rapids-build-env`? if so, when? also, what about all the other packages from `rapids-build-env`? do they need to be re-installed? it looks like now we're only reinstalling numba below.

Reply:
I'm working on the updates to the integration repo now to publish a new meta-pkg with numba 0.48. This commit will be reverted once I have the new meta-pkgs built.

Reply:
This is still a WIP PR, I copied the labels over but forgot to add it